spec/features/heroku_spec.rb

```diff
@@ -1,58 +1,73 @@
 require "spec_helper"
 
 RSpec.describe "Heroku" do
-  before do
-    drop_dummy_database
-    remove_project_directory
-  end
+  context "--heroku" do
+    before(:all) do
```

Comment:
Why move this from a `before :each` to a `before :all`? I feel like that would cause problems.